utils/freifunk-berlin-migration/uci-defaults/freifunk-berlin-01-migration.sh

```diff
@@ -553,6 +559,7 @@ migrate () {
     r1_1_0_update_uplink_notunnel_name
     r1_1_0_remove_olsrd_garbage_collection
     r1_1_0_remove_advanced_firewall
+    r1_1_0_pingcheck
```

Comment:
Do we need a migration?
Migration will only run, when the current firmware is newer than the installed config-files. as till now, there was no pingcheck package installed, there will be no configuration that needs to be migrated.

Reply:
I wasn't sure if it would set up a new package's config fine by running an upgrade or not.  

On a frest install, first the default config for pingcheck is saved in /etc/config/pingcheck.  Then at some time during the boot process the uci-defaults script is called setting up the interface and ping address to check.

By an upgrade, I'm not so sure what will happen.  
* Does the default config file from the pingcheck package get installed in /etc/config/pingcheck (a file which was not in the sysupgrade backup)?  
* Afterwards, does the uci-defaults script get called?  Notice, there isn't a guard.  Should there be?

Reply:
migrations has been removed